cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -0,0 +1,347 @@
+import numpy as np
+from typing import List, Set, Tuple, Sequence, Dict, Any
+
+from matplotlib import pyplot as plt
+from cirq import devices, ops, circuits, sim
+
+
+class CrossEntropyResult:
+    """Results from a cross-entropy benchmarking (XEB) experiment."""
+
+    def __init__(self, num_cycle_range: Sequence[int],
+                 xeb_fidelities: Sequence[float]):
+        """
+        Args:
+            num_cycle_range: The different numbers of cycles (circuit depth) at
+                which the measurements were taken.
+            xeb_fidelities: The average XEB fidelity at each number of cycles.
+        """
+        self._num_cycles = num_cycle_range
+        self._xeb_fidelities = xeb_fidelities
+
+    @property
+    def data(self) -> Sequence[Tuple[int, float]]:
```

Comment:
Give this tuple a name (`NamedTuple` from `typing`).

Reply:
Done.